app.py

```diff
@@ -203,18 +203,35 @@ def client_send_stats():
     })
 
 
+def recover_identity(f):
+    @wraps(f)
```

Comment:
Can we re-use this decorator for `signed_payload`? Or maybe this will be done in some later task?

Reply:
This will be done later.